packages/@remirror/extension-events/src/events-extension.ts

```diff
@@ -23,8 +37,13 @@ export interface EventsOptions {
  * TODO - add more events based on user feedback.
  */
 @extensionDecorator<EventsOptions>({
-  handlerKeys: ['blur', 'focus'],
-  handlerKeyOptions: { blur: { earlyReturnValue: true }, focus: { earlyReturnValue: true } },
+  handlerKeys: ['blur', 'focus', 'mousedown', 'mouseup'],
+  handlerKeyOptions: {
+    blur: { earlyReturnValue: true },
+    focus: { earlyReturnValue: true },
+    mousedown: { earlyReturnValue: true },
+    mouseup: { earlyReturnValue: true },
```

Comment:
Wasn't sure if the `earlyReturnValue` applies here?

Reply:
It does, so your code is correct 👍

Reply:
FYI `earlyReturnValue` means that if any of the registered handlers returns this value, it doesn't run any of the lower priority handlers. It can be set to be a predicate function as well.